mirrord/cli/src/error.rs

```diff
@@ -236,6 +236,13 @@ pub(crate) enum CliError {
     #[error("Failed to prepare mirrord operator client certificate: {0}")]
     #[diagnostic(help("{GENERAL_BUG}"))]
     OperatorClientCertError(String),
+
+    #[error("mirrord operator was not found in the cluster.")]
+    #[diagnostic(help(
+        "Used command requires a mirrord operator or operator usage was explicitly enabled in the configuration file.
```

Comment:
```suggestion
        "Command requires the mirrord operator or operator usage was explicitly enabled in the configuration file.
```

Something like this?